notebooks/bayes3d_paper/run_ycbv_evaluation.py

```diff
@@ -160,12 +173,19 @@ def run_tracking(scene=None, object=None, debug=False):
                 key, choicemap, (hyperparams, previous_state)
             )[0]
 
-            from b3d.chisight.gen3d.inference import inference_step
+            import b3d.chisight.gen3d.inference as inference
+            import b3d.chisight.gen3d.inference_old as inference_old
+            import b3d.chisight.gen3d.settings
+
+            inference_hyperparams = b3d.chisight.gen3d.settings.inference_hyperparams
 
             ### Run inference ###
             for T in tqdm(range(len(all_data))):
                 key = b3d.split_key(key)
-                trace = inference_step(trace, key, all_data[T]["rgbd"])
+                trace = inference.advance_time(key, trace, all_data[T]["rgbd"])
+                trace = inference_old.inference_step(trace, key, inference_hyperparams)[
```

Comment:
@nishadgothoskar for tonight, can we aim to launch at least 1 run that is using the new inference algorithm?  This will be through a call to `inference.inference_step_c2f`.  I can add this to the script if helpful.

Reply:
yes definitely. that is the plan.